hedera-node/src/main/java/com/hedera/services/stats/HapiOpSpeedometers.java

```diff
@@ -86,13 +90,18 @@ public HapiOpSpeedometers(
 				lastHandledTxnsCount.put(function, 0L);
 			}
 		});
+		receivedDeprecatedTxns = new StatsSpeedometer(halfLife);
+		lastReceivedDeprecatedTxnCount = 0L;
 	}
 
 	public void registerWith(Platform platform) {
 		registerSpeedometers(platform, receivedOps, SPEEDOMETER_RECEIVED_NAME_TPL, SPEEDOMETER_RECEIVED_DESC_TPL);
 		registerSpeedometers(platform, submittedTxns, SPEEDOMETER_SUBMITTED_NAME_TPL, SPEEDOMETER_SUBMITTED_DESC_TPL);
 		registerSpeedometers(platform, handledTxns, SPEEDOMETER_HANDLED_NAME_TPL, SPEEDOMETER_HANDLED_DESC_TPL);
 		registerSpeedometers(platform, answeredQueries, SPEEDOMETER_ANSWERED_NAME_TPL, SPEEDOMETER_ANSWERED_DESC_TPL);
+
+		platform.addAppStatEntry(speedometer.from(SPEEDOMETER_RECEIVED_DEPRECATED_NAME_TPL,
+				SPEEDOMETER_RECEIVED_DEPRECATED_DESC_TPL, receivedDeprecatedTxns));
```

Comment:
nit : similar to the suggestion above for `HapiOpCounters.registerCounters()`

Reply:
Fixed @anighanta  !